src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abbreviationaswordinname/InputAbbreviationAsWordInNameType7.java

```diff
@@ -0,0 +1,46 @@
+/*
+AbbreviationAsWordInName
+allowedAbbreviations = ORDER, OBSERVATION, UNDERSCORE, TEST
+
+
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abbreviationaswordinname;
+
+public class InputAbbreviationAsWordInNameType7 {
+    // violation below 'Abb.* in name 'HBCKL_LOCK_PATH' must contain no .* than '4' .*cap.* let.*.'
```

Comment:
Use style like
name 'HBCKL_LOCK_PATH' must contain no more than '4' .*cap`
The less `.*` the better.

Reply:
Done